setting_up_firely_server/configuration/mongodb/db_mongo_auth.rst

```diff
@@ -305,8 +305,10 @@ Once access control is enabled, you have to configure the user and password in t
 
 .. note:: 
 
-    Currently, only a single connectionstring can be configured for MongoDB. The roles and users above differentiate between the authorization needed to perform an automatic upgrade, and the authorization needed for regular operation.
-    You may choose to use a connectionstring with the ``fs_data_upgrade_user`` only when performing an upgrade, and afterwards reset it to the ``fs_data_user``.
+    Currently, there is an option to configure two connectionstring for MongoDB both Data database and Administration database. 
+    The roles and users above differentiate between the authorization needed to perform an automatic upgrade, and the authorization needed for regular operation.
+    You may use a connectionstring with the ``fs_data_upgrade_user`` in `AutoUpdateConnectionString` property that will be used only when performing an upgrade, and `ConnectionString` with the ``fs_data_user`` for normal use.
+    If `AutoUpdateConnectionString` is not set, the `ConnectionString` will be used for both.
```

Comment:
I would rewrite this a bit:

"The roles and users above differentiate between the authorization needed to perform an automatic upgrade, and the authorization needed for regular operation of Firely Server. It is possible to configure separate connection strings for these two roles and this can be done for both the administration database and the regular repository database. 
You may use the ``fs_data_upgrade_user`` in the `AutoUpdateConnectionString` field that will be used only when performing an upgrade of the MongoDb schemas, and the ``fs_data_user`` in the regular `ConnectionString` field for normal use. If `AutoUpdateConnectionString` is not set, the `ConnectionString` will be used in both cases."

Maybe you could also add an example of the app setting configuration, such as this snippet:

    "MongoDbOptions": {
        "ConnectionString": "mongodb://fs_data_user:fs_data_secret@localhost:27017/vonkdata?authSource=fs_data",
        "EntryCollection": "vonkentries",
        "MaxLogLine": 300,
        "AutoUpdateConnectionString" : "mongodb://fs_data_upgrade_user:fs_data_upgrade_secret@localhost:27017/vonkdata?authSource=fs_data"
    },